lib/tasks/add_packages.rake

```diff
@@ -0,0 +1,9 @@
+task add_packages: :environment do
```

Comment:
create a single rake file with name one_time_2019.rake 
and write all rake tasks in that file

Reply:
done